Mutational Signatures Methods (#68)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

_MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.
The fusion filtering workflow can be found in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering).

### Mutational Signatures

> Using consensus SNVs, we obtained weights for signature sets using deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] with BSgenome.Hsapiens.UCSC.hg38 annotation [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].

Reviewer: ```suggestion
We obtained weights for signature sets by applying deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] to consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
```

> We used each sample's signature weights to estimate how many mutations for each sample contributed to each signature.

Reviewer: ```suggestion
We estimated how many mutations contributed to each signature for each sample 
 using each sample's signature weights.
```

> Weights are normalized from 0 to 1.

Reviewer: ```suggestion
We normalized weights from 0 to 1 with a linear transformation.
```

Reviewer: I think this is what you mean. Unless you are saying that the property of the weights is that they are distributed from zero to one.

Author: The second thing you said. This is not a transformation I applied but I thing that is already a property of the weights obtained from deconstructSigs. But I felt understanding this helps explain the rationale for the number of contributing mutations estimation.

Reviewer: Ok - see if what I wrote seems ok.

> For a given sample and signature:  

Reviewer: What is this calculating? I think you could clarify by adding a bit more to this sentence fragment.

$$ \frac{\textrm{# of contributing mutations}}{Mb} = \frac{\textrm{weight} * \sum{\textrm{# Trinucleotide mutations}}} {\textrm{Size in Mb of effectively surveyed genome}}
$$

> Note that these reports do not take into account signatures of small contribution; deconstructSigs drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].

Reviewer: ```suggestion
These results do not include signatures with small contributions; deconstructSigs drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].
```

> These methods were employed for both COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.

Reviewer: ```suggestion
We used these methods to calculate signature scores for each sample with both COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.
```
